HomeWork_4/src/Helper.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * The class contain some <code>String</code> constants for user info, help
+ * and some errors. It contain command line commands for program.
+ *
+ * @author Nikuta Zenchik
+ */
+public class Helper {
```

Comment:
Inappropriate class name - Helper implies that it does some work and helps, but it contains only consts. something like Messages would be more clear.